cmd/trivyproxy/main.go

```diff
@@ -108,7 +109,9 @@ func (a *API) proxyToTrivy(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	stdout, stderr, err := a.runTrivy(r.Context(), imageURL)
+	keppelToken := r.Header.Get(trivy.KeppelToken)
```

Comment:
This does not read very well: `trivy.KeppelToken` sounds like it would contain a token value, instead of the name of the token header.

```suggestion
	keppelToken := r.Header.Get(trivy.KeppelTokenHeader)
```